internal/scanner/scanner.go

```diff
@@ -57,26 +69,91 @@ func (s *Scanner) ScanNamespace(namespace string) error {
 
 	// TODO this is for debugging, it should be remove in future steps!
 	for _, resource := range auditableResources {
-		fmt.Println("Policies: ")
+		log.Debug().Msg("Policies: ")
 		for _, policy := range resource.Policies {
-			fmt.Println(policy.GetName())
+			url, err := s.resourcesFetcher.GetPolicyServerURLRunningPolicy(context.Background(), policy)
+			if err != nil {
+				log.Debug().Err(err).Msg("CANNOT GET POLICY SERVER URL")
+				continue
+			}
+			log.Debug().Msgf("POLICY SERVER URL: %s", url.String())
 		}
 		for _, resource := range resource.Resources {
-			fmt.Println(resource)
-			fmt.Println("........")
+			log.Debug().Msgf("Opa!")
+			log.Debug().Dict("resource", zerolog.Dict().
+				Str("name", resource.GetName()).
+				Str("kind", resource.GetKind()).
+				Str("namespace", resource.GetNamespace()).
+				Str("UID", string(resource.GetUID()))).
+				Msg("auditable resource")
 		}
-		fmt.Println("---------------------")
 	}
 
-	// TODO for next steps:
 	// Iterate through all auditableResources. Each item contains a list of resources and the policies that would need
 	// to evaluate them. You need to create the AdmissionReview request using the resource which is an unstructured.
 	// unstructured nested functions might help with that https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured
 	// or the UnstructuredContent() method that returns a map
+	for i := range auditableResources {
```

Comment:
Nit: the part of the comment where it says "create AdmissionReview" onwards could go as godoc of `auditResource()`.